src/drake/core.clj

```diff
@@ -106,8 +106,13 @@
    it. This is safe to do since it's the default filesystem,
    but it gives us a bit easier compatibility with existing tools."
   [filename]
-  (let [n (dfs/normalized-path filename)]
```

Comment:
That's a pretty awesome idea. I'll go ahead and pop that in there when I make my modifications for your review comments.

I'll probably do this late night tonight.